app/src/main/java/com/gh4a/resolver/BrowseFilter.java

```diff
@@ -11,11 +11,13 @@
 import com.gh4a.utils.IntentUtils;
 
 public class BrowseFilter extends AppCompatActivity {
+    private static final String EXTRA_INITIAL_COMMENT = "initial_comment";
+
     public static Intent makeRedirectionIntent(Context context, Uri uri,
             IntentUtils.InitialCommentMarker initialComment) {
         Intent intent = new Intent(context, BrowseFilter.class);
         intent.setData(uri);
-        intent.putExtra("initial_comment", initialComment);
+        intent.putExtra(EXTRA_INITIAL_COMMENT, initialComment);
```

Comment:
We might want to share this constant. It feels a bit weird to have it used only in a single place.

Reply:
I've added a commit that moves the intent extra access to BrowseFilter and shares the constant.